src/native/http_request_response.c

```diff
@@ -289,7 +289,10 @@ static int s_on_incoming_headers_fn(
     size_t num_headers,
     void *user_data) {
 
-    (void)block_type;
+    if (block_type != AWS_HTTP_HEADER_BLOCK_MAIN) {
```

Comment:
What's this about? Could you add a comment if this is intentional?

Reply:
Uh, oops, lemme push that up to Java.